doc/snippets/Microsoft.Data.SqlClient/SqlCommand.xml

```diff
@@ -1,3108 +1,4330 @@
+        </para>
+      </remarks>
+      <example>
+        <para>
+          The following example creates a <see cref="T:Microsoft.Data.SqlClient.SqlCommand" /> and sets some of its properties.
+        </para>
+        <!-- SqlCommand_SqlCommand2 -->
+        <code language="c#">
+          using System;
+          using System.Data;
+          using Microsoft.Data.SqlClient;
+
+          namespace SqlCommandCS
+          {
+              class Program
+              {
+                  static void Main()
+                  {
+                      string str = "Data Source=(local);Initial Catalog=Northwind;"
+                          + "Integrated Security=SSPI";
+                      string qs = "SELECT OrderID, CustomerID FROM dbo.Orders;";
+                      CreateCommand(qs, str);
+
+                  }
+
+                  private static void CreateCommand(string queryString, string connectionString)
+                  {
+                      using (SqlConnection connection = new SqlConnection(connectionString))
+                      {
+                          SqlCommand command = new SqlCommand(queryString, connection);
+                          connection.Open();
+                          SqlDataReader reader = command.ExecuteReader();
+                          while (reader.Read())
+                          {
+                              Console.WriteLine(String.Format("{0}, {1}", reader[0], reader[1]));
+                          }
+                      }
+                  }
+              }
+          }
+        </code>
+      </example>
+    </ctor>
+    <ctor name="cmdTextStringAndSqlConnectionAndSqlTransaction">
+      <param name="cmdText">
+        The text of the query.
+      </param>
+      <param name="connection">
+        A <see cref="T:Microsoft.Data.SqlClient.SqlConnection" /> that represents the connection to an instance of SQL Server.
+      </param>
+      <param name="transaction">
+        The <see cref="T:Microsoft.Data.SqlClient.SqlTransaction" /> in which the <see cref="T:Microsoft.Data.SqlClient.SqlCommand" /> executes.
+      </param>
+      <summary>
+        Initializes a new instance of the <see cref="T:Microsoft.Data.SqlClient.SqlCommand" /> class with the text of the query, a <see cref="T:Microsoft.Data.SqlClient.SqlConnection" />, and the <see cref="T:Microsoft.Data.SqlClient.SqlTransaction" />.
+      </summary>
+      <remarks>
+        <para>
+          The following table shows initial property values for an instance of <see cref="T:Microsoft.Data.SqlClient.SqlCommand" />.
+        </para>
+        <para>
+          <list type="table">
+            <listheader>
+              <term>Properties</term>
+              <description>Initial value</description>
+            </listheader>
+            <item>
+              <term><see cref="P:Microsoft.Data.SqlClient.SqlCommand.CommandText" /></term>
+              <description><c>cmdText</c></description>
```

Comment:
Is this even a valid 'Initial value'?

Reply:
🤷‍♂️ I'm not checking accuracy of the documentation, just fixing the structure (and already going way above just changing markdown blocks to xml)